modules/Sources/Generated/L10n.swift

```diff
@@ -484,6 +484,14 @@ public enum L10n {
     public static let succeeded = L10n.tr("Localizable", "send.succeeded", fallback: "Sending transaction succeeded")
     /// Send Zcash
     public static let title = L10n.tr("Localizable", "send.title", fallback: "Send Zcash")
+    public enum Error {
+      /// insufficient funds
+      public static let insufficientFunds = L10n.tr("Localizable", "send.error.insufficientFunds", fallback: "insufficient funds")
+      /// invalid address
+      public static let invalidAddress = L10n.tr("Localizable", "send.error.invalidAddress", fallback: "invalid address")
+      /// invalid amount
+      public static let invalidAmount = L10n.tr("Localizable", "send.error.invalidAmount", fallback: "invalid amount")
```

Comment:
Our [issue](https://github.com/zcash/secant-android-wallet/issues/340) also mentions a few more errors we could handle. I didn't analyze how much we can align them with what the SDK provides, but I rather put the question here.

Reply:
I think the `fee` feedback looks interesting. Other 4 are probably covered on our side. If the fee is agreed to do, we will doit in a follow up ticket.